akka-persistence/src/main/scala/akka/persistence/Eventsourced.scala

```diff
@@ -504,7 +504,7 @@ private[persistence] trait Eventsourced extends Snapshotter with PersistenceStas
 
       case RecoveryTick(true) ⇒
         try onRecoveryFailure(
-          new RecoveryTimedOut(s"Recovery timed out, didn't get snapshot within $timeout s"),
+          new RecoveryTimedOut(s"Recovery timed out, didn't get snapshot within $timeout"),
```

Comment:
So the difference is that PrettyDuration renders in the "largest, complete" time unit, for example:

```
scala> 100.seconds.toString
res9: String = 100 seconds

scala> PrettyDuration.format(100.seconds)
res10: String = 1.667 min
```

No strong feelings either way in this message about which style is better